config/prow/config.yaml

```diff
@@ -67,6 +67,7 @@ deck:
     '*':
       github_team_ids:
       - 2009231 # test-infra-admins
+      - 2460384 # milestone-maintainers
```

Comment:
I'm going to merge this for now because it isn't 100% obvious to me why this shouldn't be more broadly accessible. However can you consider scoping this down to the job/jobs you manage rather than everything?

Reply:
Moving bigger discussion to https://github.com/kubernetes/test-infra/issues/17321